md/txdata_intf.md

```diff
@@ -0,0 +1,12 @@
+| Name                       | Signed? | Packed Dim      | Unpacked Dim   | Clock Domain | JTAG Dir | Reset Val                                    |
```

Comment:
This set of registers controls the logic that produces data for the TX.  It has four modes: PRBS, constant, square wave, and pulse.